<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicCloudFileManagerDialog</class>
 <widget class="QWidget" name="MusicCloudFileManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件管理</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>520</width>
     <height>369</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>文件管理</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>520</width>
     <height>344</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(255,255,255,150);}
QLabel{background:transparent;}</string>
   </property>
  </widget>
  <widget class="MusicCloudUploadTableWidget" name="uploadTableWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>84</y>
     <width>480</width>
     <height>270</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reuploadButton">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>44</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重传</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>uploadTableWidget</zorder>
  <zorder>reuploadButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicCloudUploadTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">musiccloudfilemanagerdialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
